clusterloader2/testing/density/config.yaml

```diff
@@ -0,0 +1,128 @@
+automanagedNamespaces: 3
+steps:
+- measurements:
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: SchedulerLatency
+    Method: SchedulerLatency
+    Params:
+      action: reset
+      provider: gce
+      masterName: e2e-ksiedlecki2-master
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+- phases:
+  - namespaceRange:
+      min: 0
+      max: 2
+    replicasPerNamespace: 1
+    tuningSet: TuningSet100
+    objectBundle:
+    - basename: rc1
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        Replicas: 30
+- measurements:
+  - Identifier: WaitForRunningPods1
+    Method: WaitForRunningPods
+    Params:
+      desiredPodCount: 90
+      labelSelector: phase = phase1
+      timeout: 2m
+- measurements:
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: stop
+      label: Pods creation
+  - Identifier: TimerAdditionals
+    Method: Timer
+    Params:
+      action: start
+- phases:
+  - namespaceRange:
+      min: 0
+      max: 2
+    replicasPerNamespace: 1
+    tuningSet: TuningSet5
+    objectBundle:
+    - basename: rc1-add
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        Replicas: 1
+- measurements:
+  - Identifier: WaitForRunningPods2
+    Method: WaitForRunningPods
+    Params:
+      desiredPodCount: 93
+      labelSelector: phase = phase1
+- measurements:
```

Comment:
IIUC we are still not measuring pod startup time, right?

Reply:
You are right. It's not implemented yet.